packages/api/src/module/write/learning-materials-tasks/domain/complete-task.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { CompleteTask } from '@/module/commands/complete-task.domain-command';
+import { TaskWasCompleted } from '@/module/events/task-was-completed.domain-event';
+
+import { completeTask } from './complete-task';
+
+describe('complete task', () => {
```

Comment:
For me there is lack of test: 
what if task with ID 1 was completed, but now i want to complete task with ID 2.